src/routes/RemediationDetails.js

```diff
@@ -47,6 +48,26 @@ class RemediationDetails extends Component {
         this.props.switchAutoReboot(this.id, autoReboot);
     };
 
+    handleSuccessBanner = (id, name) => {
+        // TODO: Needs to check when playbook is done
+        this.props.toggleExecutePlaybookBanner();
+        this.props.addNotification({
+            variant: 'success',
+            title: `Remediation plan ${name} successfully completed.`,
+            dismissDelay: 8000
+        });
+    }
+
+    handlePlaybookCancel = (id, name) => {
+        // TODO: Cancel playbook
+        this.props.toggleExecutePlaybookBanner();
+        this.props.addNotification({
+            variant: 'danger',
```

Comment:
Is danger correct in here? Since this will be shown everytime user closes playbook execution I would make this warning instead of danger.

Reply:
Yeah, this is when a user cancels a playbook. This wasn't in the mocks, but it's something that we want. I'll talk to UXD to figure it out!

Reply:
>>> 
Katie Riker  2 minutes ago
good idea. Should be "info" toast (not a success, not a warning, not an error).
With a 2 second think: "Canceled execution of playbook [foo]" -- but let me circle back on this. I'll add to mocks